### PR TITLE
Fix numeric check for tilitid values

### PR DESCRIPTION
**Pull request by signedav**

### CODE REVIEW

File: modelbaker/dbconnector/mssql_connector.py [hunk 1]
```diff
@@ -984,8 +985,23 @@ def create_basket(self, dataset_tid, topic, tilitid_value=None):
                 if not tilitid_value:
                     # default value
                     tilitid_value = "NEWID()"
-                elif not tilitid_value.isnumeric():
+                elif not isinstance(tilitid_value, numbers.Number):
                     tilitid_value = f"'{tilitid_value}'"
+                print(
```
Comment (domi4484) [inline modelbaker/dbconnector/mssql_connector.py:990]: Forgotten debug print?

File: modelbaker/dbconnector/pg_connector.py [hunk 2]
```diff
@@ -1020,11 +1021,28 @@ def create_basket(self, dataset_tid, topic, tilitid_value=None):
                     topic
                 )
             try:
+                print(tilitid_value)
```
Comment (domi4484) [inline modelbaker/dbconnector/pg_connector.py:1024]: Here as well

**Review by domi4484 — APPROVED** [review]

File: modelbaker/dbconnector/pg_connector.py (hunk 2 above)
Comment (signedav) [reply modelbaker/dbconnector/pg_connector.py:1024]: check if you are doing it right ;-) 

No, I forgot. Thanks.

File: modelbaker/dbconnector/mssql_connector.py [hunk 3]
```diff
@@ -984,8 +985,23 @@ def create_basket(self, dataset_tid, topic, tilitid_value=None):
...
+                        schema=self.schema,
+                        sequence="t_ili2db_seq",
+                        tid_name=self.tid,
+                        tilitid_name=self.tilitid,
+                        basket_table=BASKET_TABLE,
+                        dataset_tid=dataset_tid,
+                        topic=topic,
+                        tilitid=tilitid_value,
+                    )
+                )
```
Comment (signedav) [inline modelbaker/dbconnector/mssql_connector.py:1004]:
```suggestion
```

File: modelbaker/dbconnector/pg_connector.py (hunk 2 above)
Comment (signedav) [inline modelbaker/dbconnector/pg_connector.py:1024]:
```suggestion
```

File: modelbaker/dbconnector/pg_connector.py [hunk 4]
```diff
@@ -1023,8 +1024,24 @@ def create_basket(self, dataset_tid, topic, tilitid_value=None):
...
+                        schema=self.schema,
+                        sequence="t_ili2db_seq",
+                        tid_name=self.tid,
+                        tilitid_name=self.tilitid,
+                        basket_table=PG_BASKET_TABLE,
+                        dataset_tid=dataset_tid,
+                        topic=topic,
+                        tilitid=tilitid_value,
+                    )
+                )
```
Comment (signedav) [inline modelbaker/dbconnector/pg_connector.py:1044]:
```suggestion
```

**Pull request merged**